aiida/tools/importexport/dbexport/__init__.py

```diff
@@ -138,241 +365,225 @@ def export(
         raise exceptions.ArchiveExportError(f"The output file '{filename}' already exists")
 
     if silent:
-        logging.disable(level=logging.CRITICAL)
+        logging.disable(logging.CRITICAL)
+
+    writer = writers[file_format](filename=filename, use_compression=use_compression)
+
+    summary(
+        writer.file_format_verbose,
+        filename,
+        include_comments=include_comments,
+        include_logs=include_logs,
+        **traversal_rules
+    )
+
+    output_data = {}
```

Comment:
If I understand correctly, this dictionary just contains timings of the different steps 

```suggestion
    export_timings = {}
```

Reply:
as mentioned above, I guess just any data helpful for "post-analysis"